ic-os/components/setupos-scripts/hardware.sh

```diff
@@ -251,7 +251,8 @@ function verify_deployment_path() {
         echo -e "\n\n\n\n\n\n"
         echo -e "\033[1;31mWARNING: Gen2 hardware detected but no Node Operator Private Key found\033[0m"
         echo -e "\033[1;31mGen2 hardware should be onboarded using the Gen2 Node Deployment method\033[0m"
-        echo -e "\033[1;31mPlease redeploy using the Gen2 Node Deployment method\033[0m"
+        echo -e "\033[1;31mIf you already onboarded this node machine with an HSM and are now just *redeploying* the node, you may continue (just wait 5 minutes for the installation to resume)\033[0m"
```

Comment:
Language nit, does this sound simpler?
> If you are redeploying a node machine that currently uses an HSM, please wait for the installation to resume.

Reply:
I just update the wording based on a sasa message:
`Okay so how can a node provider who uses an HSM today deploy a new node without an HSM?
`
Now the warning reads:
```
WARNING: Gen2 hardware detected but no Node Operator Private Key found.
Gen2 hardware should be deployed using the Gen2 Node Deployment method.

If you already completed your *Node Provider onboarding* using the legacy procedure (with an HSM),
you may continue your HSM node deployment (just wait 5 minutes for the installation to resume).
```